x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts_list.ts

```diff
@@ -368,19 +368,22 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await refreshAlertsList();
         await find.waitForDeletedByCssSelector('.euiBasicTable-loading');
         const refreshResults = await pageObjects.triggersActionsUI.getAlertsListWithStatus();
-        expect(refreshResults.map((item: any) => item.status).sort()).to.eql(['Error', 'Ok']);
+        expect(refreshResults.map((item: any) => item.status).sort()).to.eql([
+          'Failed',
+          'Succeeded',
+        ]);
       });
       await refreshAlertsList();
       await find.waitForDeletedByCssSelector('.euiBasicTable-loading');
-      await testSubjects.click('ruleExecutionStatusFilterButton');
```

Comment:
@JiaweiWu and I talked more about it and we think we should add a new config file to allow this test working when the feature flag is off. We will create an issue to add back this test until the feature flag exists